syntax="proto3";
package csproto;

message LoginRequest {
    int32 user_id = 1;
    string passwd = 2;
}

message LoginResponse {
    bool success = 1;
    int64 now = 2;
    string session = 3;
}